05-UI-ReactJS/source/js/components/views/movie-list.js

```diff
@@ -0,0 +1,49 @@
+import React, { PropTypes } from "react";
+import { Link } from "react-router";
+
+export default class MovieList extends React.Component {
+  render() {
+    if(!this.props.movies.length) {
```

Comment:
In order to clean this, you could have two different methods to render and do this:
`return (this.props.movies.length) ? renderList() : renderEmptyBlock()`